server/api/user.js

```diff
@@ -12,3 +12,44 @@ router.get('/', requireToken, async (req, res, next) => {
     next(err);
   }
 });
+
+router.post('/', async (req, res, next) => {
```

Comment:
right now this route is `POST /api/user` 
which is usually used to ADD a user rather than associating a doki and a user